cl/recap/api_serializers.py

```diff
@@ -314,6 +314,19 @@ def validate(self, attrs):
                 "Cannot use 'docket_number' parameter "
                 "without 'court' parameter."
             )
+
+        if (
+            attrs.get("pacer_case_id")
+            and is_appellate_court(attrs.get("court").pk)
+            and not attrs.get("docket_number")
+        ):
```

Comment:
You might as well flip this logic to get early exit without doing the DB hit that `is_appellate_court` does. So:

```suggestion
        if (
            attrs.get("pacer_case_id")
            and not attrs.get("docket_number")
            and is_appellate_court(attrs.get("court").pk)
        ):
```

Reply:
Thanks! I'll apply the suggestion.